src/api/lib/activexml/node.rb

```diff
@@ -220,7 +220,7 @@ def element_name=(name)
       _data.name = name
     end
 
-    def _data # nodoc
```

Comment:
Sidenote, it's not even a valid :nodoc:, but just a comment

Reply:
yes :wink: but even if ti was used at some point it doesn't make any sense :wink:

Reply:
I didn't question that it doesn't make sense;-)